src/master/master.cfg

```diff
@@ -141,15 +146,15 @@ c['schedulers'] = [
                      },
                      hour=[0, 6, 12, 18],
                      minute=[5]),
-  schedulers.Nightly(name='Semi-daily (local macOS builds)',
+  schedulers.Nightly(name='Daily (local macOS builds)',
```

Comment:
If the machine is fast enough, twice daily would be great :)

I was thinking about suggesting the Q4 goal for Safari stable+TP latency (from push to results) to be 24 hours, and that the average latency is 1.5 x cycle time.

Reply:
> If the machine is fast enough, twice daily would be great :)

Unfortunately, the machine is not fast enough :/ More cause for more machines

> I was thinking about suggesting the Q4 goal for Safari stable+TP latency (from push to results) to be 24 hours, and that the average latency is 1.5 x cycle time.

This build runs once a day, but it includes both releases of Safari. That sounds like what you're describing here, though, so I think I'm misreading something. Would you mind re-phrasing?

Reply:
Starting a run once every 24 hours that takes 12 hours to complete means that the latency from PR being merge to results being available is in the 12-36 hour range. To get the worst-case latency to 24 hours it's necessary to run more often.

Although for simplicity we could pick a cadence (daily or 12h) and measure what percentage of those commits did get complete results.